perfkitbenchmarker/configs/default_config_constants.yaml

```diff
@@ -14,6 +14,10 @@ default_single_core: &default_single_core
     machine_type: m3.medium
     zone: us-east-1
     image: null
+  SoftLayer:
+    machine_type: '{"cpus": 4, "memory": 4096, "os": "UBUNTU_LATEST_64", "nic": 1000}'
```

Comment:
It looks like you're using a four core machine as the `default_single_core` for Softlayer.   Is this intentional?